src/utilities/validators/parameter-validator.ts

```diff
@@ -64,7 +65,12 @@ class ParameterValidator extends BaseValidator {
     example: Json,
     path: string[],
   ): void {
-    if (parameterHasSchema(parameter) && !parameterHasContent(parameter)) {
+    if (parameter.example && parameter.examples) {
```

Comment:
This was a pretty simple addition, but the thing that gets me is that we're checking this _after_ the example groups are created, which means we would be creating a nonsense example group if this fails. I was thinking maybe we split the parameter validator into 2 parts, one that validates the schema itself, and then a second that validates the example groups against the schema. We could move the content/schema checks to the parameter schema validator as well. Interested in people's opinions on this

Reply:
@RockslideJr at first glance, I would vote for the splitting of the parameter validation into 2 parts.  That might have advantages and allow more flexibility for the future.

Reply:
I agree, I also like the idea of splitting this into two parts!